client/cli/go/cmds/cluster.go

```diff
@@ -43,10 +47,22 @@ func init() {
 			"\n\tdefault. Use '--file=false' to disable creation of file"+
 			"\n\tvolumes on this cluster.")
 
+	clusterSetFlagsCommand.Flags().StringVar(&cl_block_str, "block", "",
+		"\n\tOptional: Control the possibility of creating block volumes"+
```

Comment:
Perhaps "Control the possibility of" is more succinctly written as "Allow" or "Permit"

Reply:
Actually, thinking about it, the text is not *that* wrong:

The `--block` option lets you specify whether or not it should be possible to create block volumes on the cluster. It lets us allow or forbid (i.e. control) that possibility. That's where I was coming from.

I still agree that it's not worded too well. Thinking how to frame it better...